### DISCUSSION TITLE
How to set value to id?

### DISCUSSION THREAD

**Discussion by alucardu** [prisma/prisma#9686] [Q&A] [answered]
I have this model in my schema:

```prisma
model Movie {
  id             Int            @id @default(autoincrement())
  original_title String         @unique
  poster_path    String         @unique
  release_date   String
  users          User[]
  Notification   Notification[]
}
```

I want to assign a id from my client to the id field of the model.

**Comment by ryands17** [comment, marked as answer]
@alucardu 👋 

You already have `@default(autoincrement())` setup so you cannot assign an id from your client. If you'd like to perform this, then you can remove the default from the `id` field.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
